locale/en/get-involved/events.md

```diff
@@ -5744,13 +5744,21 @@ regions:
   - region: Latin America
     conferences:
       - name: NodeConf Argentina
-        desc: 'The first international Node.js conference in Latin American territory.'
+        desc: 'NodeConf Argentina is among the first international Node.js conferences taking place in Latin American territory.'
         dates:
           - 2016-11-18T11:00:00.000Z
           - 2016-11-19T11:00:00.000Z
         link: 'https://2016.nodeconf.com.ar/'
         lat: -34.6066339
         lon: -58.4110953
+      - name: NodeConf Brazil
+        desc: 'This is the first edition of NodeConf in Latin America and is also the first major event focused exclusively on Node.js of Brazil.'
```

Comment:
Well it is the text copied from the website, and posted in the issue
discussion. You are correct, it does sound odd along with thoughts of other
options : 'from Brazil', 'of Brazil', 'in Brazil'...
On Jun 1, 2016 02:45, "Luigi Pinca" <notifications@github.com> wrote:

> In locale/en/get-involved/events.md
> <https://github.com/nodejs/nodejs.org/pull/770#discussion_r65305155>:
>
> >          dates:
> >            - 2016-11-18T11:00:00.000Z
> >            - 2016-11-19T11:00:00.000Z
> >          link: 'https://2016.nodeconf.com.ar/'
> >          lat: -34.6066339
> >          lon: -58.4110953
> > +      - name: NodeConf Brazil
> > +        desc: 'This is the first edition of NodeConf in Latin America and is also the first major event focused exclusively on Node.js of Brazil.'
>
> I'm not a native speaker but "focused exclusively on Node.js of Brazil"
> sounds a bit weird, should it be "focused exclusively on Node.js in Brazil"?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/nodejs/nodejs.org/pull/770/files/0e1898de40c1946502d941e1f1e178d544bd1a10#r65305155>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAMAeRZgEjk3VHIrgq0y1iY7SlhPIhVcks5qHRxpgaJpZM4IrHds>
> .
>